project/plugins.sbt

```diff
@@ -0,0 +1,7 @@
+// protoc related plugin and dependencies must be loaded before the Lagom plugin.
+addSbtPlugin("com.thesamet" % "sbt-protoc" % "0.99.8")
+libraryDependencies += "com.trueaccord.scalapb" %% "compilerplugin" % "0.5.47"
```

Comment:
This here is biggets problem so far: `sbt-protoc` must be loaded before the lagom plugin otherwise underlying layers of `sbt-protoc` (actually `proto-bridge`) will pick the wrong version of `protobuf-java` (it neeeds `3.2.0` but obtains `2.6.1`)  and code generation will fail.